### PR TITLE
A few corrections to docs and the CLI interface

### PR DESCRIPTION
**Pull request by urimandujano**
- Fixes a malformed JSON document that wasn't rendering correctly on readthedocs
- Updates a key in an example
- Updates CLI command descriptions

### CODE REVIEW

**Comment by urimandujano** [comment]
@jimPruyne bump on taking a look at this if you can. The meat is probably making sure that the CLI command descriptions still make sense but overall is a pretty small change.

File: docs/source/using_cli_and_flows.rst [hunk 1]
```diff
@@ -105,7 +105,7 @@ If the command is formatted properly, the resulting output will look like the fo
 
   {
     "action_id": "<An id>",
-    "status": "ACTIVE"
+    "status": "ACTIVE",
```
Comment (jimPruyne) [inline docs/source/using_cli_and_flows.rst:108]: nice catch

File: examples/flows/hello-world/definition.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
             "ActionScope": "https://auth.globus.org/scopes/actions.globus.org/hello_world",
             "Parameters": {
                 "echo_string.$": "$.input.echo_string",
-                "sleep.$": "$.input.sleep"
+                "sleep_time.$": "$.input.sleep"
```
Comment (jimPruyne) [inline examples/flows/hello-world/definition.json:13]: another good one!

File: globus_automate_client/cli/flows.py [hunk 3]
```diff
@@ -405,15 +421,19 @@ def flow_actions_list(
...
-        PROD_FLOWS_BASE_URL, hidden=True, callback=flows_endpoint_envvar_callback,
+        PROD_FLOWS_BASE_URL,
+        hidden=True,
+        callback=flows_endpoint_envvar_callback,
     ),
     verbose: bool = verbosity_option,
 ):
     """
-    List the Actions associated with a Flow.
+    List a Flow definition's discrete invocations.
```
Comment (jimPruyne) [inline globus_automate_client/cli/flows.py:436]: :+1:

**Review by jimPruyne — APPROVED** [review]
good catches and clarifications!

**Pull request merged**